### ISSUE TITLE
Information Stick not working on Armor Stands???

### ISSUE THREAD

**Issue by CaptainCyanite17** [Serilum/.issue-tracker#231]
Minecraft version: 1.16.1
Forge version: 32.0.108
Environment: Singleplayer

Mod name: Entity Information
Mod version: entityinfo_1.16-1.3

So I tried to use the information stick on an armor stand, but all it does is destroy the armor stand without giving information. Is this because of a bug, or is it something you havent figured out how to incorporate yet? No rush, all I'm trying to do is give the armor stand some arms without using the /summon command, because when I do it that way, the arms work, but no matter what I try, the armor stand always faces south and I can' rotate it. That's why I downloaded this mod, but don't worry I'm not disappointed in it, it's still very useful. Please let me know why this happens

thanks!!!

{Images}

![2020-08-18_17 00 29](https://user-images.githubusercontent.com/69873613/90565417-e30da680-e174-11ea-9ae8-3989e8e1307b.png)

![2020-08-18_17 03 23](https://user-images.githubusercontent.com/69873613/90565442-ed2fa500-e174-11ea-93ed-1a3e232eb61a.png)

![2020-08-18_17 02 42](https://user-images.githubusercontent.com/69873613/90565459-f1f45900-e174-11ea-9c69-2a9bc9b94990.png)

**Comment by ricksouth** [comment]
I appreciate you using the mod, but unfortunately the use you're trying it for is not the intended one. It is more to get information on Mob Entities in order to make them compatible while in the development process. I'll think about some sort of armour stand mod!

**Comment by CaptainCyanite17** [comment]
Thanks, maybe I can help? I'm not an expert with making mods but it would be fun to try. Also the world you see in the images is actually going to be a multiplayer server called PvParkour. It started out with size pvp arenas and one mob arena, then quickly expanded once I opened it as a LAN. I added the parkour part, then a market/shop to buy things, then a teams pvp, and lastly an elytra run. Unfortunately the original world was lost, so this is the newer & better version: bigger + more things to do I don't know why I told you all this, but maybe it will help people know that the server is going to exist
